src/apps/bm_devkit/sdi12_example/CMakeLists.txt

```diff
@@ -0,0 +1,336 @@
+    ${SRC_DIR}/lib/drivers/htu21d.cpp
+    ${SRC_DIR}/lib/drivers/ms5803.cpp
+    ${SRC_DIR}/lib/drivers/ina232.cpp
+    ${SRC_DIR}/lib/drivers/pca9535.c
+    ${SRC_DIR}/lib/drivers/protected/protected_i2c.c
+    ${SRC_DIR}/lib/drivers/protected/protected_spi.c
+    ${SRC_DIR}/lib/drivers/abstract/abstract_spi.cpp
+    ${SRC_DIR}/lib/drivers/stm32_adc.c
+    ${SRC_DIR}/lib/drivers/stm32_io.c
+    ${SRC_DIR}/lib/drivers/stm32_rtc.c
+    ${SRC_DIR}/lib/drivers/tca9546a.cpp
+    ${SRC_DIR}/lib/lwip/lwip_support.c
+    ${SRC_DIR}/lib/drivers/w25.cpp
+    ${SRC_DIR}/lib/bm_serial/bm_serial.c
+    ${SRC_DIR}/lib/memfault/memfault_platform_core_u5.c
+    ${SRC_DIR}/lib/sensor_sampler/sensorSampler.cpp
+    ${SRC_DIR}/lib/sensor_sampler/pressureSampler.cpp
+    ${SRC_DIR}/lib/sensor_sampler/htuSampler.cpp
+    ${SRC_DIR}/lib/sensor_sampler/powerSampler.cpp
+    ${SRC_DIR}/lib/bristlefin/bristlefin.cpp
+    ${SRC_DIR}/lib/common/LineParser.cpp
+    ${SRC_DIR}/lib/common/OrderedSeparatorLineParser.cpp
+    ${SRC_DIR}/lib/common/Exo3LineParser.cpp
+    ${SRC_DIR}/third_party/aligned_malloc/aligned_malloc.c
+    ${SRC_DIR}/third_party/crc/crc16.c
+    ${SRC_DIR}/third_party/crc/crc32.c
+    ${SRC_DIR}/third_party/fnv/hash_32a.c
+    ${SRC_DIR}/third_party/fnv/hash_64a.c
+    ${SRC_DIR}/third_party/FreeRTOS-Plus-CLI/FreeRTOS_CLI.c
+    ${SRC_DIR}/third_party/mbedtls_base64/base64.c
+    ${SRC_DIR}/third_party/printf/printf.c
+    ${SRC_DIR}/third_party/tinycbor/src/cborparser.c
+    ${SRC_DIR}/third_party/tinycbor/src/cborencoder_float.c
+    ${SRC_DIR}/third_party/tinycbor/src/cborencoder.c
+    ${SRC_DIR}/third_party/tinycbor/src/cborerrorstrings.c
+    ${SRC_DIR}/third_party/tinycbor/src/cborvalidation.c
+    ${SRC_DIR}/lib/drivers/bme280driver.cpp
+    ${SRC_DIR}/third_party/BME280_driver/bme280.c
+    )
+
+set(LIB_INCLUDES
+    ${SRC_DIR}/lib/bcmp
+    ${SRC_DIR}/lib/cli
+    ${SRC_DIR}/lib/common
+    ${SRC_DIR}/lib/debug
+    ${SRC_DIR}/lib/drivers
+    ${SRC_DIR}/lib/drivers/adin2111/include
+    ${SRC_DIR}/lib/drivers/include
+    ${SRC_DIR}/lib/drivers/protected
+    ${SRC_DIR}/lib/drivers/abstract
+    ${SRC_DIR}/lib/lwip
+    ${SRC_DIR}/lib/bm_serial
+    ${SRC_DIR}/lib/memfault
+    ${SRC_DIR}/lib/middleware
+    ${SRC_DIR}/third_party/
+    ${SRC_DIR}/third_party/aligned_malloc/
+    ${SRC_DIR}/third_party/crc
+    ${SRC_DIR}/third_party/fnv/
+    ${SRC_DIR}/third_party/FreeRTOS-Plus-CLI/
+    ${SRC_DIR}/third_party/printf/
+    ${SRC_DIR}/lib/bridge
+    ${SRC_DIR}/lib/bristlefin
+    ${SRC_DIR}/lib/sensor_sampler
+    ${SRC_DIR}/lib/sys
+    ${SRC_DIR}/third_party/tinycbor/src
+    ${SRC_DIR}/third_party/BME280_driver
+    )
+
+#
+# MCUBoot files/includes
+#
+set(MCUBOOT_DIR ${SRC_DIR}/third_party/mcuboot)
+set(MCUBOOT_FILES
+    ${MCUBOOT_DIR}/boot/bootutil/src/boot_record.c
+    ${MCUBOOT_DIR}/boot/bootutil/src/bootutil_misc.c
+    ${MCUBOOT_DIR}/boot/bootutil/src/bootutil_public.c
+    # ${MCUBOOT_DIR}/boot/bootutil/src/caps.c
```

Comment:
@uma-kat, did you comment these out, or were these commented out lines copy-pasted from a different cmake file?

Reply:
@evanShap I don't recollect editing this `CmakeLists.txt` file. Could these have been auto generated and added here? :o

Reply:
@evanShap I copied the CMakeLists.txt from serial_payload app. Lmk if I need to change this, otherwise could approve this PR if it looks alright?